catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -274,7 +258,16 @@
                             <goal>grunt</goal>
```

Comment:
I think it's so we can skip the default grunt build, otherwise the build would fail because there is no longer a `Gruntfile.js`.